tracking_manager/tests/test_tracking_manager.py

```diff
@@ -36,6 +36,10 @@ def setUpClass(cls):
         )
         cls.partner_model = cls.env.ref("base.model_res_partner")
         cls._active_tracking(["bank_ids", "category_id"])
+        bank_ids_field = cls.partner_model.field_id.filtered(
+            lambda x: x.name == "bank_ids"
+        )
+        bank_ids_field.custom_tracking = True
```

Comment:
needed to pass tests due to field config:

[{'__last_update': datetime.datetime(2024, 8, 27, 14, 25, 44, 120107),
'column1': False,
'column2': False,
'complete_name': False,
'compute': False,
'copied': False,
'create_date': False,
'create_uid': False,
'custom_tracking': False,
'depends': False,
'display_name': 'Banks (res.partner)',
'domain': False,
'field_description': 'Banks',
'group_expand': False,
'groups': [],
'help': False,
'id': 930,
'index': False,
'model': 'res.partner',
'model_id': (79, 'Contact'),
'modules': 'base',
'name': 'bank_ids',
'native_tracking': False,
'on_delete': False,
'readonly': False,
'related': False,
'related_field_id': False,
'relation': 'res.partner.bank',
'relation_field': 'partner_id',
'relation_field_id': False,
'relation_table': False,
'required': False,
'selectable': True,
'selection': False,
'selection_ids': [],
'size': 0,
'state': 'base',
'store': True,
'trackable': True,
'tracking': 0,
'translate': False,
'ttype': 'one2many',
'write_date': datetime.datetime(2024, 8, 27, 14, 25, 44, 120107),
'write_uid': (1, 'OdooBot')}]